src/components/layout/Post.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+	<div class="post">
+		<div class="post__voters">
+			<arrow-icon
+				class="post__voters-arrow"
+				:class="{ 'post__voters-vote': vote }"
+			/>
+			{{ voters }}
+		</div>
+		<div class="post__content">
+			<h3 class="post__content-title">{{ title }}</h3>
+			<p class="post__content-bodyMarkdown" v-html="bodyMarkdown" />
```

Comment:
Try converting markdown to HTML and then use something like `htmlData.replace(/(<([^>]+)>)/ig, "");` to remove all tags. IK you wanna do it later. Just leaving a suggestion.

Reply:
Technically this will be some kind of short description of around 20 to 30 words.